apps/dashboard/app/(app)/mobile-sidebar.tsx

```
@@ -30,36 +45,32 @@ export const MobileSideBar = ({ className }: Props) => {
+                      {item.label}
+                    </Button>
+                  </SheetClose>
+                </Link>
+              ))}
+            </div>
+
+            <Separator className="my-2" />
+            <h2 className="px-2 mb-2 text-lg font-semibold tracking-tight">Resources</h2>
+            <div className="space-y-1">
+              {resourcesNavigation.map((item) => (
+                <Link href={`${item.href}`} target="_blank">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)